department-of-veterans-affairs/caseflow-efolder
# Anya/701 manifests controller v2 part2

connects #701 

This is a second part of the PR:
1. Implemented some of the TODO comments from the PR #1
2. Wrote tests

Things to do (I will create tickets):
1. Make the job asynchronous and handle pending state
2. Implement start caching records from VBMS to S3 once manifest is fetched. 

Steps to Validate:
1. 
```Curl the API with curl "http://localhost:3000/api/v2/manifests" -H "Authorization: Token token=token" -H "css_id: CF_Q_283" -H "station_id: 283" -H "veteran_id: 1234"```
2. Ensure statuses are correct
3. Ensure the record list is accurate
